doc/src/conf.py

```diff
@@ -65,6 +65,8 @@
 # Usually you set "language" from the command line for these cases.
 language = None
 
+highlight_language = 'none'
```

Comment:
Hmm. Eventually this should be `'redprl'`. Consider this as a bug.

Reply:
Agreed, but let's not worry about this quite yet.